app/routes/public/cfs/edit-session.js

```diff
@@ -15,7 +15,9 @@ export default Route.extend({
       }),
       session: await this.store.findRecord('session', params.session_id, {
         include: 'session-type,track'
```

Comment:
@shreyanshdwivedi Isn't track and session already included with session? To avoid two extra network requests can we utilise something like `session.track` or similar?

Reply:
@uds5501 IMO the session-type and track here represents the one related to session. 
Whereas the one I added will fetch all the session-types and tracks related to event (for dropdown).
Don't you think so?

Reply:
@shreyanshdwivedi I see your point. You are right

Reply:
@shreyanshdwivedi but the issue was resolved just by doing this? Nothing else was required in the form?

Reply:
@uds5501 yes. For me it is working fine now. i'll add a screenshot to PR description.
However I'll appreciate if you can pull changes and review this PR. @kushthedude you too :)